js_modules/dagit/packages/core/src/configeditor/codemirror-yaml/mode.tsx

```diff
@@ -693,12 +693,9 @@ CodeMirror.registerHelper(
     let lastMarkLocation: CodeMirror.Position | undefined;
 
     yamlDoc.errors.slice(0, 10).forEach((error) => {
-      const from = codeMirrorDoc.posFromIndex(
-        error.source?.range ? error.source.range.start : 0,
-      ) as CodeMirror.Position;
-      const to = codeMirrorDoc.posFromIndex(
-        error.source?.range ? error.source.range.end : Number.MAX_SAFE_INTEGER,
-      ) as CodeMirror.Position;
+      const [fromPos, toPos] = error.pos;
```

Comment:
Hmm I think this is accounting for the fact that some errors don't specify a range and we want them to appear as "full document" errors instead of nowhere? Not sure if this is addressed by the yaml upgrade?

Reply:
Based on the types for `YAMLError`, it looks like the `pos` tuple should be non-null. This makes me think that the issue is resolved in the new version.

![image](https://user-images.githubusercontent.com/2823852/149827012-b3b361e4-c3d8-4cd2-b1aa-42161e716d2e.png)